packages/@lwc/template-compiler/src/__tests__/fixtures/static-content/key-in-slot/actual.html

```diff
@@ -0,0 +1,8 @@
+<template>
+    <slot>
+        <div for:each={defaultItems} for:item="item" key={item}>Default fallback</div>
+    </slot>
+    <slot name="foo">
+        <div for:each={fooItems} for:item="item" key={item}>Named fallback</div>
+    </slot>
+</template>
```

Comment:
This is from a test case that kept breaking as I was writing this PR:

https://github.com/salesforce/lwc/blob/989d8107f25602602d3ae3536b983d32cf6edf44/packages/%40lwc/integration-karma/test/rendering/slotting/index.spec.js#L41

This test confirms that we make the key unique enough to handle adjacent slots with different names – one default and one non-default. This is handled by `generateKeyExpression` and `slotParentName` in `codegen/index.ts`.